### PR TITLE
Fix modal height - Increase Css specificity 

### PR DESCRIPTION
**Pull request by MAvaisAshraf**
checked the issue we used class and due to low specificity this code was not overwriting, I have increased Specificity by adding !important;

### CODE REVIEW

**Comment by HarelM** [comment]
Can you please resolve the conflict? 
Also can you use height < ... Instead of max-height (similar to the other media query)?

**Comment by MAvaisAshraf** [comment]
Resolved Conflict and changed max-height to height <= as requested in the media query.

File: IsraelHiking.Web/src/application/components/sidebar/sidebar.component.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.title {
```
Comment (HarelM) [inline IsraelHiking.Web/src/application/components/sidebar/sidebar.component.scss:1]: Please revert these changes in this file.

**Comment by HarelM** [comment]
Thanks! can you please share a screenshot of how this looks after the fix?

**Comment by MAvaisAshraf** [comment]
note: Add dummy text just for screenshots to increase modal height to see the scroll effect.
modal-horizontal
modal-vertical

**Review by HarelM — APPROVED** [review]
